api/resource.rb

```diff
@@ -151,6 +154,18 @@ def validate
       end
     end
 
+    # Represents a list of documentation links.
+    class IamPolicy < Api::Object
+      # boolean of if this binding should be generated
+      attr_reader :exclude
+
+      def validate
+        super
+
+        check :exclude, type: :boolean, default: true
```

Comment:
Super minor / not a blocker at all, but I'd vote for making this default to false; individual providers can exclude the field if they don't support it yet / don't want to support it.